### PR TITLE
Remove all {{Draft}} macro calls in Markdown files

### LINKED ISSUE #4876: > Deprecate and remove Draft kumascript macro
> Yeah, we still have several hundred calls to {{Draft}}, although I think we have consensus for removing them.

Shall we close this one and spin up a new issue specifically for phasing out `Draft`?

_Originally posted by @schalkneethling in https://github.com/mdn/yari/issues/2844#issuecomment-960502902_

### PR DESCRIPTION
**Pull request by tannerdolby**
#### Summary

After reading through #10273, it seemed the general consensus was to begin removing all `{{Draft}}` macro calls from any templates. This PR removes all remaining `{{Draft}}` occurrences.

#### Motivation

To help resolve the discussion in 10273.

#### Related issues

Fixes [#4876](https://github.com/mdn/yari/issues/4876) in Yari (still a few references to `{{Draft}}` in the Draft.test.js file)
Related #10273 

#### Metadata

- [ ] Adds a new document
- [x] Rewrites (or significantly expands) a document - (Removes content from templates)
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/mdn/structures/page_types/api_reference_page_template/index.md [hunk 1]
```diff
@@ -51,15 +51,15 @@ browser-compat: path.to.feature.NameOfTheInterface
...
 
 - `\{{APIRef("<em>GroupDataName</em>")}}` — this generates the left hand reference sidebar showing quick reference links related to the current page. For example, every page in the [WebVR API](/en-US/docs/Web/API/WebVR_API) has the same sidebar, which points to the other pages in the API. To generate the correct sidebar for your API, you need to add a GroupData entry to our KumaScript GitHub repo, and include the entry's name inside the macro call in place of _GroupDataName_. See our [API reference sidebars](/en-US/docs/MDN/Contribute/Howto/Write_an_API_reference/Sidebars) guide for information on how to do this.
-- `\{{Draft}}` — this generates a **Draft** banner that indicates that the page is not yet complete, and should only be removed when the first draft of the page is completely finished. After it is ready to be published, you can remove this.
+- `\` — this generates a **Draft** banner that indicates that the page is not yet complete, and should only be removed when the first draft of the page is completely finished. After it is ready to be published, you can remove this.
```
Comment (wbamberg) [inline files/en-us/mdn/structures/page_types/api_reference_page_template/index.md:54]:
```suggestion
```

...let's just delete the whole thing here

Comment (tannerdolby) [reply files/en-us/mdn/structures/page_types/api_reference_page_template/index.md:54]: Ok, will do.

**Review by wbamberg — APPROVED** [review]
👍 this is awesome @tannerdolby . Thanks!

**Comment by tannerdolby** [comment]
You're welcome @wbamberg! I will submit a PR to remove the lowercase `{{draft}}` occurrences shortly :)

**Pull request merged**